etc/inc/disable-programs.inc

```diff
@@ -822,6 +827,7 @@ blacklist ${HOME}/.cache/INRIA
 blacklist ${HOME}/.cache/MusicBrainz
 blacklist ${HOME}/.cache/NewsFlashGTK
 blacklist ${HOME}/.cache/QuiteRss
+blacklist ${HOME}/.cache/Quotient/quaternion
```

Comment:
Would it make sense to `blacklist ${HOME}/.cache/Quotient`?

Reply:
Same reason as the /usr/share/quotient one...https://github.com/quotient-im/libQuotient